internal/management/repository/eventsourcing/view/org.go

```diff
@@ -0,0 +1,60 @@
+package view
+
+import (
+	"github.com/caos/zitadel/internal/errors"
+	"github.com/caos/zitadel/internal/model"
+	org_model "github.com/caos/zitadel/internal/org/model"
+	org_view "github.com/caos/zitadel/internal/org/repository/view"
+	"github.com/caos/zitadel/internal/view"
+)
+
+const (
+	orgTable = "management.orgs"
+)
+
+func (v *View) OrgByID(orgID string) (*org_view.OrgView, error) {
+	return org_view.OrgByID(v.Db, orgTable, orgID)
+}
+
+func (v *View) OrgByDomain(domain string) (*org_view.OrgView, error) {
+	orgs, _, err := org_view.SearchOrgs(v.Db, orgTable, &org_model.OrgSearchRequest{
+		Limit: 1,
+		Queries: []*org_model.OrgSearchQuery{
+			{
+				Key:    org_model.ORGSEARCHKEY_ORG_DOMAIN,
+				Method: model.SEARCHMETHOD_EQUALS,
+				Value:  domain,
+			},
+		}})
+	if err != nil {
+		return nil, err
+	}
+	if len(orgs) == 0 {
+		return nil, errors.ThrowNotFound(nil, "VIEW-ByecF", "no org found")
+	}
+	return orgs[0], nil
```

Comment:
we should discuss how we like to handle this, just that its everywhere the same (eg UserByEmail)